packages/components/notification/src/notify.ts

```diff
@@ -170,7 +170,17 @@ export function closeAll(): void {
   }
 }
 
+export function updateOffsets(position: NotificationOptions['position']) {
```

Comment:
`position` can be given a default value.

https://github.com/element-plus/element-plus/blob/a5582a78f3545265c3ae7e8bf36c05104770a919/packages/components/notification/src/notification.ts#L90

This method needs to be synchronized in the document.